### PR TITLE
Refactored to beta3, Restructured Package.md

### PR DESCRIPTION
**Pull request by TharinduNM**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,74 +1,57 @@
...
+# SCIM Connector
+ 
+ SCIM Connector provides a Ballerina API to access the Rest API of any service that has implemented SCIM2 specification.
+ It handles [OAuth2](https://tools.ietf.org/html/rfc6749).
+ 
+ 
+ ## Compatibility
+ | Ballerina Language Version| SCIM API Version  |
+ | :------------------------:| :----------------:|
+ | 0.970.0-beta1             | SCIM2.0           |
```
Comment (Biruntha) [inline README.md]: Change version to 0.970.0-beta3

File: README.md [hunk 2]
```diff
@@ -1,74 +1,57 @@
...
+
+ ## Getting Started
+  1. Refer https://ballerina.io/learn/getting-started/ to download Ballerina and install tools.
```
Comment (muthulee) [inline README.md]: Put Getting Started here.

File: scim2/Package.md [hunk 3]
```diff
@@ -1,64 +1,59 @@
...
- | ------------- |:-------------:| -----:|
- | ballerina-0.970.0-beta1-SNAPSHOT     | 0.9.6 | SCIM2.0 |
- 
-
-![alt text](resources/SCIM2.png)
-
-The source code of the SCIM2 endpoint can be found at [package-scim2](https://github.com/wso2-ballerina/package-scim2)
+ | Ballerina Language Version| SCIM API Version  |
+ | :------------------------:| :----------------:|
+ | 0.970.0-beta1             | SCIM2.0           |
```
Comment (Biruntha) [inline scim2/Package.md]: Change version to 0.970.0-beta3

File: scim2/Package.md [hunk 4]
```diff
@@ -1,64 +1,59 @@
-# Ballerina SCIM Endpoint
-
-*The system for Cross-domain Identity Management (SCIM) specification
- is designed to make managing user identities in cloud-based applications 
- and services easier.*
-
+# SCIM Connector
```
Comment (Biruntha) [inline scim2/Package.md]: Change to "SCIM Endpoint"

File: scim2/Package.md [hunk 5]
```diff
@@ -1,64 +1,59 @@
...
  
- The Ballerina SCIM2 Endpoint allows users to access Rest API of any service that implements 
- SCIM2 specification.  
+ SCIM Connector provides a Ballerina API to access the Rest API of any service that has implemented SCIM2 specification.
```
Comment (Biruntha) [inline scim2/Package.md]: Change to SCIM Endpoint

File: scim2/tests/README.md [hunk 6]
```diff
@@ -1,15 +1,15 @@
-# Ballerina SCIM Endpoint - Tests
+# Ballerina SCIM Connector - Tests
```
Comment (Biruntha) [inline scim2/tests/README.md]: Change to "SCIM Endpoint"

File: scim2/tests/README.md [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
 The following sections provide you with information on how to use the Ballerina SCIM Endpoint.
 
-## Compatibility
-| Language Version        | Endpoint Version          | API Versions  |
-| ------------- |:-------------:| -----:|
-| ballerina-0.970.0-beta1-SNAPSHOT     | 0.9.6** | SCIM2.0 |
+ ## Compatibility
+ | Ballerina Language Version| SCIM API Version  |
+ | :------------------------:| :----------------:|
+ | 0.970.0-beta1             | SCIM2.0           |
```
Comment (Biruntha) [inline scim2/tests/README.md]: Change version as 0.970.0-beta3

File: scim2/Package.md (hunk 4 above)
Comment (TharinduNM) [reply scim2/Package.md]: I think it should be Connector

File: scim2/Package.md (hunk 5 above)
Comment (TharinduNM) [reply scim2/Package.md]: I think it should be connector

**Pull request merged**